### ISSUE TITLE
Direct Deposit page - iOS is unable to select editable fields with VoiceOver enabled

### ISSUE THREAD

**Issue by digitalken** [department-of-veterans-affairs/va.gov-team#21338]
## What happened?
iPhone will only select the entire fields (Routing number, Account number, Account Type) with VoiceOver enabled.  I'm unable to select only the editable field (the input) so that I can enter the numbers.

iOS only, OK with Android.

## Specs:
- Device: iPhone XR.
(Using account judy.morrison)

## Steps to Reproduce
- Enable VoiceOver on iPhone and attempt to select only the editable fields.

## Desired behavior
User is able to edit the fields with VoiceOver enabled by selecting only the input fields.

## Acceptance Criteria
User is able to edit the fields with VoiceOver enabled by selecting only the input fields.

## Bug Severity
- 2 - High

## Linked to Story
https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/20222

## Screen shot(s) and additional information

![image.png](https://images.zenhubusercontent.com/5f6a83c84093730d05b5cee6/22f32f46-ec3b-4e1b-9a76-e44f66999f63)

**Comment by digitalken** [comment]
PASS 3/26/2021 - Looks good on iOS.  Closing.
